hudi-client/hudi-client-common/pom.xml

```diff
@@ -38,7 +38,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hudi</groupId>
-      <artifactId>hudi-aws</artifactId>
```

Comment:
so it was only due to cloud watch metrics we had hudi-client-common depend on hudi-aws is it? For log configurations, we don't have any dependencies?